js/models/autocomplete.es6.js

```diff
@@ -0,0 +1,30 @@
+const Parent = window.DDG.base.Model;
+
+function Autocomplete (attrs) {
+
+    Parent.call(this, attrs);
+
+};
+
+
+Autocomplete.prototype = $.extend({},
+  Parent.prototype,
+  {
+
+      modelName: 'autocomplete',
```

Comment:
Every model needs a unique `modelName` attribute now. It will throw an error if you don't give it one.